<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>429</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>19</y>
      <width>191</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导弹基础伤害</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>目标半径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导弹爆炸半径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导弹爆炸速度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>目标速度</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLineEdit" name="DaoDanJiChuShangHai">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="MuBiaoBanJin">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DaoDanBaoZhaBanJin">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DaoDanBaoZhaSuDu">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="MuBiaoSuDu">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="drfBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>---超大型巡航导弹---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  超大型巡航导弹(全部类型) 0.882</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>---超大型鱼雷---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  超大型鱼雷(全部类型) 1.00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>---火箭---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  火箭 0.644</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  狂乱火箭 0.882</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  标枪火箭 0.682</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>---屹立导弹---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  屹立导弹(全部类型) 0.944</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>---轻型导弹---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  轻型导弹 0.604</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  精准轻型导弹 0.561</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  愤怒轻型导弹 0.682</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  自动锁定轻型导弹 0.604</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>---巡航导弹---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  巡航导弹 0.882</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  精准巡航导弹 0.735</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  愤怒巡航导弹 0.908</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  自动锁定巡航导弹 0.882</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>---鱼雷---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  鱼雷 0.944</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  狂乱鱼雷 0.967</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  标枪鱼雷 0.967</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>---重型导弹---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  重型导弹 0.682</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  精准重型导弹 0.583</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  愤怒重型导弹 0.882</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  自动锁定重型导弹 0.682</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>---重型攻击导弹---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  重型攻击导弹 0.882</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  愤怒重型攻击导弹 0.920</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  标枪重型攻击导弹 0.895</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>---电弧投射器---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  小型电弧投射器 0.4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  中型电弧投射器 0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  大型电弧投射器 0.7</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计算</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>最终造成伤害：</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ZuiZhongZaoChengShangHai">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Num_drf">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>伤害衰减系数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>制作：by政委（wdnmd QRZ）</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
